src/bundle/ui-dev/src/modules/sub-items/sub.items.module.js

```diff
@@ -880,72 +879,48 @@ export default class SubItemsModule extends Component {
                 'ibexa_sub_items',
             );
         }
+        const confirmBtn = {
```

Comment:
it's more of `confirmBtnAttrs` now, as it's not button per se
Also, in all other declarations below as well